crates/turborepo-lib/src/cli/mod.rs

```diff
@@ -811,6 +830,63 @@ fn path_non_empty(s: &str) -> Result<Utf8PathBuf, String> {
     }
 }
 
+#[derive(Copy, Clone, Debug, PartialEq)]
+struct ContinueModeParser;
+
+impl ContinueModeParser {
+    pub fn new() -> Self {
+        Self {}
+    }
+}
+
+// slightly redundant, but this allows us to transform deprecated values
```

Comment:
I'm torn on whether this is too much. Some alternatives include:
  - a simpler `PossibleValuesParser` that maps to correct enum values, emitting deprecation warnings. cons: true / false appear in the generated help text. but we could just hide all of them via `hide_possible_values`
  - include deprecated values in the enum and just use `ValueEnum`. cons: true / false appear in the generated help text, and downstream logic is more complicated

Reply:
We can make it a simple `PossibleValuesParser`. Since `2.0.0` `--continue=true|false` has not been accepted so we do not need to support them or emit deprecation warnings.